File: Modul-4/Soal-1/PRAK401-2410817210016-MUHAMMADRIZKYAKBAR.c
#include <stdio.h>

int main()
{
    int mtp, i;
    char pgt;

    printf("Kelipatan: ");
    scanf("%d", &mtp);
    printf("Pengganti: ");
    scanf(" %c", &pgt);

    for (i = 1; i <= 50; i++) {
        if (i % mtp == 0){
            printf("%c ", pgt);
            } else {
                printf("%d ", i);
            }

        if (i % 10 == 0){
            printf("\n");
            }
    }

return 0;
}

File: Modul-4/Soal-1/PRAK401-2410817210016-MUHAMMADRIZKYAKBAR.py
mtp = int(input("Kelipatan: "))
pgt = input("Pengganti: ")

for i in range (1,51):
    if i % mtp == 0:
        print(pgt, end=" ")
    else:
        print(i, end=" ")

    if i % 10 == 0:
        print()